model/metrics/v8js-metrics.yaml

```diff
@@ -0,0 +1,84 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
+
+  - id: metric.veightjs.memory.size
+    type: metric
+    metric_name: v8js.memory.size
+    brief: "External memory size."
```

Comment:
This metric is a little bit confusing. The name didn't reflect it is a figure for external memory size.

Reply:
Updated description